app/mailers/prison_mailer.rb

```diff
@@ -107,4 +107,8 @@ def smoke_test?
   def visitors_email_address
     visit.visitors.first.email
   end
+
+  def formatted_date
+    Date.parse(@visit.slots.first.date).strftime('%A %e %B')
```

Comment:
I think I'd go one further and make this a named format (see http://api.rubyonrails.org/classes/DateTime.html#method-i-to_formatted_s and https://github.com/ministryofjustice/peoplefinder/blob/master/config/locales/en.yml#L22-L25 for examples).